### PR TITLE
[Missing Tokens] Write Directly to Spellbook File

### PR DESCRIPTION
**Pull request by bh2smith**
just set the `SPELLBOOK_PATH` env var and away you go!

```sh
export SPELLBOOK_PATH=
cd $SPELLBOOK_PATH
git checkout main && git pull && git checkout -b token-update
python -m src.missing_tokens
```

Once we merge this it will be:

```
docker run \
    -e SPELLBOOK_PATH=<YOUR_SPELLBOOK_PATH> \
    -e INFURA_KEY=<YOUR_INFURA_KEY> \
    -e DUNE_API_KEY=<YOUR_DUNE_API_KEY> \
    ghcr.io/cowprotocol/data-misc-missing-tokens
```

### CODE REVIEW

File: tests/test_missing_tokens.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            self.assertEqual(file_lines, file.readlines())
+
+        for i, old_line in enumerate(file_lines):
+            new_line = f"New Line {i + 1}\n"
+            replace_line(old_line, new_line, test_file)
+
+            file_lines[i] = new_line
+            with open(test_file, "r", encoding="utf-8") as file:
+                self.assertEqual(file_lines, file.readlines())
+        os.remove(test_file)
```
Comment (gentrexha) [inline tests/test_missing_tokens.py:31]: In my opinion, separating these three different use cases into separate methods for each case makes them easier to understand and maintain. What's your take on that?

**Review by gentrexha — APPROVED** [review]
Worked like a charm! Thanks, Ben! New PR using the new code: https://github.com/duneanalytics/spellbook/pull/3444

Left a minor test suggestion.

File: tests/test_missing_tokens.py (hunk 1 above)
Comment (bh2smith) [reply tests/test_missing_tokens.py:31]: Ya sure --- this isn't really a production product, but I could go and spend more time fixing this. Could be several days before I get back to that.

Comment (bh2smith) [reply tests/test_missing_tokens.py:31]: Not gonna do that now. I need to start using this.

Comment (bh2smith) [reply tests/test_missing_tokens.py:31]: Issue: https://github.com/cowprotocol/data-misc/issues/34

**Pull request merged**